data/json/items/gun/faults_gun.json

```diff
@@ -2,13 +2,26 @@
   {
     "id": "fault_gun_blackpowder",
     "type": "fault",
-    "name": "Blackpowder fouling",
-    "description": "Firing blackpowder loads from a gun fouls it, which reduces reliability and, if left uncleaned, leads to rust. It fouls the gun much faster than the use of modern smokeless powder cartridges. Fouling is only a significant impact on reliability at high levels, but black powder fouling accumulates quickly.",
-    "time": "50 m",
+    "name": "Clean blackpowder fouling without lubricating",
+    "description": "Firing blackpowder loads from a gun fouls it, which reduces reliability and, if left uncleaned, leads to rust. It fouls the gun much faster than the use of modern smokeless powder cartridges. Fouling is only a significant impact on reliability at high levels, but black powder fouling accumulates quickly. **Cleaning your gun without lubricating it will result in an unreliable (but clean) weapon.",
+    "time": "23 m",
     "skills": [ [ "mechanics", 1 ] ],
     "requirements": {
       "qualities": [ { "id": "SCREW", "level": 1 } ],
-      "tools": [ [ [ "pipe_cleaner", -1 ], [ "small_repairkit", -1 ], [ "large_repairkit", -1 ] ] ]
+      "tools": [ [ [ "pipe_cleaner", -1 ], [ "small_repairkit", -1 ], [ "large_repairkit", -1 ] ] ],
+      "components": [
+        [
+          [ "gasoline", 5 ],
+          [ "diesel", 5 ],
+          [ "chem_ethanol", 5 ],
+          [ "denat_alcohol", 5 ],
+          [ "vinegar", 5 ],
+          [ "ammonia", 5 ],
+          [ "bleach", 5 ],
+          [ "soapy_water", 5 ]
```

Comment:
You do know these all don't have the same volume right? All .25L:
![image](https://user-images.githubusercontent.com/51099123/67448285-85580e80-f5e4-11e9-9b6e-4144946a78ed.png)
![image](https://user-images.githubusercontent.com/51099123/67448336-ad477200-f5e4-11e9-92e1-b483f623e3c3.png)